backend/services/portale_alloggi_service.py

```
@@ -0,0 +1,797 @@
+        """
+        try:
+            soap_envelope = self._create_auth_soap_envelope()
+            headers = self._create_auth_headers()
+
+            response = requests.post(self.wsdl_url, data=soap_envelope, headers=headers, timeout=30)
+            response.raise_for_status()
+
+            return self._parse_auth_response(response.text)
+
+        except requests.exceptions.RequestException as e:
+            logger.error("Network error during authentication: %s", str(e))
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)